packages/template-retail-react-app/app/pages/checkout/partials/shipping-options-only.jsx

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import {FormattedNumber, useIntl} from 'react-intl'
+import {
+    Box,
+    Flex,
+    Stack,
+    Text,
+    VStack,
+    Radio,
+    RadioGroup
+} from '@salesforce/retail-react-app/app/components/shared/ui'
+import {Controller} from 'react-hook-form'
+import {useShippingMethodsForShipment} from '@salesforce/commerce-sdk-react'
+import PropTypes from 'prop-types'
+import LoadingSpinner from '@salesforce/retail-react-app/app/components/loading-spinner'
+
+// Component to handle shipping options for a single shipment (without product cards)
+const ShippingOptionsOnly = ({shipment, basketId, currency, control}) => {
```

Comment:
Is it better to call it SingleAdressShippingOption vs MultiAddressesShippingOptions?